Move assets using Blockchain Dashboard (#372)
status-im/ift-docs: docs/blockchain/node-app/move-assets-using-logos-blockchain-app.md

---
title: Move assets using the Logos Blockchain app
doc_type: procedure
product: blockchain
topics: [blockchain-app]
steps_layout: flat
authors: danielsanchezq, kashepavadan
owner: logos
doc_version: 1
slug: move-assets-using-logos-blockchain-app
---

# Move assets using the Logos Blockchain node app

#### Get started with token transfers between wallet accounts directly from the dashboard UI.

This procedure covers how to send funds from one of your wallet addresses to a recipient on the Logos Blockchain using the Logos Blockchain dashboard. It is intended for users who need to move tokens between accounts without using the CLI or crafting transactions manually.

Before you start, make sure you have the following:

- A built [Blockchain node app](./build-and-run-logos-blockchain-node-app-ui.md)

> - At least one wallet address with a positive balance, for example one funded via the faucet at [`testnet.blockchain.logos.co`](https://testnet.blockchain.logos.co/web/faucet/)

Reviewer: **Network mismatch — still open from last round.** This links the **testnet** faucet, but the app is **devnet-only**:
- `BlockchainBackend.cpp:398` hardcodes `deployment.insert("well_known_deployment", "devnet")`
- `GenerateConfigView.qml` exposes only a **Devnet** radio (+ Custom) — no testnet option
- the app's own `README.md` funds from `https://devnet.blockchain.logos.co/web/faucet/`

Funding a testnet address against a devnet node puts the wallet on the wrong network. `https://devnet.blockchain.logos.co/web/faucet/` returns 200, so:

```suggestion
- At least one wallet address with a positive balance, for example one funded via the faucet at [`devnet.blockchain.logos.co`](https://devnet.blockchain.logos.co/web/faucet/)
```

Reviewer: Update after re-dogfooding against the current app: **this is now resolved upstream — keep the `testnet` link, don't apply the devnet suggestion.**

`logos-blockchain-ui` switched from devnet to testnet in [#32 "fix(network): default"](https://github.com/logos-blockchain/logos-blockchain-ui/commit/920d9b282140ed4be4472c018ed770b11daa9566), which landed *after* my review above. That commit:
- removes the `deployment.insert("well_known_deployment", "devnet");` hardcode in `BlockchainBackend.cpp`
- removes the `devnetRadio` from `GenerateConfigView.qml`
- repoints the README faucet + block explorer to `testnet.blockchain.logos.co`

So the app is on testnet now, this doc's `testnet.blockchain.logos.co/web/faucet/` link is correct (200 OK), and the earlier testnet→devnet suggestion would reintroduce the mismatch. I'd resolve this thread.

Also re-verified the rest of the doc end-to-end against the app source on this branch's head (`4432c83`): the Transfer panel labels (`To key (64 hex chars)`, `Amount`, `Send`), the read-only balance / `---` placeholder, the result-row copy button, and the three troubleshooting strings (`Module not initialized`, `Call failed.`, `Error: …`) all match. The fresh commit's balance-refresh fix is correct — the **Refresh** button does live in the Accounts panel (`AccountsView.qml`), not on the From-address combo.

_Out of scope for this PR, but flagging for follow-up:_ the prerequisite `build-and-run-logos-blockchain-node-app-ui.md` still sends readers to the **devnet** faucet at Step 3 while the rest of that doc says testnet — worth realigning it to testnet too.

1. In the **Node** tab, start the node and wait until the status reads **Running**.

   - Stopping the node mid-session disables the **Operations** tab and returns the UI to the **Node** tab.

1. Click **Operations** in the top navigation, then click **Transfer** in the sidebar.

   - The **Transfer funds** panel appears.

1. Select a sender address in the **From address** dropdown.

   - The address's current balance is shown read-only next to the dropdown.
   - A balance that could not be fetched is displayed as `---`.

1. Enter the recipient's public key in the **To key (64 hex chars)** field.

1. Enter the amount to send in the **Amount** field.

1. Click **Send**.

   {% hint style="warning" %}
   Transfers are irreversible; double-check the 64-hex recipient key and the amount before pressing **Send**.
   {% endhint %}

## Read the transfer result

1. Check the result row directly below the **Send** button:

   - On success, the row displays the transaction ID or hash returned by the module. Click the copy button to copy it to the clipboard.
   - On failure, the row displays `Error: <message>` with the rejection reason from the module.

> 1. Confirm the sender's balance decreased by refreshing the **From address** dropdown.

Reviewer: **The From-address dropdown has no refresh — the Refresh button is in the Accounts panel.** In `TransferView.qml` the "From address" combo just renders `accountsModel`; there's no refresh affordance on it. Balances are re-fetched by the **Refresh** button in the **Accounts** view (`AccountsView.qml:48` → `refreshAccountsRequested()` → `backend.refreshAccounts()`). A transfer also doesn't auto-refresh (`transferFunds()` never calls `refreshAccounts()`; the only auto-refresh is the `QTimer::singleShot` after **Start Node**). So readers who "refresh the dropdown" will see a stale balance.

```suggestion
1. Confirm the sender's balance decreased by pressing **Refresh** in the **Accounts** panel.
```
